File: AppGenerateFiles/dbo/Model/Transaction_Contratos_Inversionistas.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Transaction_Contratos_Inversionistas : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? numero_cont { get; set; }
       public DateTime? fecha { get; set; }
       public Double? taza { get; set; }
       public Double? monto_inicial { get; set; }
       public string? nombre_sustituto { get; set; }
       public string? identificacion_sustituto { get; set; }
       public string? direccion_sustituto { get; set; }
       public string? departamento_sus { get; set; }
       public string? municipio_sustituto { get; set; }
       public int? id_inversor { get; set; }
       public DateTime? fecha_pago { get; set; }
       public DateTime? fecha_ultimo_pago { get; set; }
       public Double? saldo { get; set; }
       public Double? montointeres { get; set; }
       public Double? interes { get; set; }
       public DateTime? fecha_restructura { get; set; }
       public int? Id_User { get; set; }
       [ManyToOne(TableName = "Security_Users", KeyColumn = "Id_User", ForeignKeyColumn = "Id_User")]
       public Security_Users? Security_Users { get; set; }
       [ManyToOne(TableName = "Catalogo_Inversores", KeyColumn = "id_inversor", ForeignKeyColumn = "id_inversor")]
       public Catalogo_Inversores? Catalogo_Inversores { get; set; }
   }
}

File: AppGenerateFiles/dbo/FrontModel/Transaction_Contratos_Inversionistas.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { Security_Users }  from './Security_Users.js'
import { Catalogo_Inversores }  from './Catalogo_Inversores.js'
class Transaction_Contratos_Inversionistas extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ numero_cont;
   /**@type {Date}*/ fecha;
   /**@type {Number}*/ taza;
   /**@type {Number}*/ monto_inicial;
   /**@type {String}*/ nombre_sustituto;
   /**@type {String}*/ identificacion_sustituto;
   /**@type {String}*/ direccion_sustituto;
   /**@type {String}*/ departamento_sus;
   /**@type {String}*/ municipio_sustituto;
   /**@type {Date}*/ fecha_pago;
   /**@type {Date}*/ fecha_ultimo_pago;
   /**@type {Number}*/ saldo;
   /**@type {Number}*/ montointeres;
   /**@type {Number}*/ interes;
   /**@type {Date}*/ fecha_restructura;
   /**@type {Security_Users} ManyToOne*/ Security_Users;
   /**@type {Catalogo_Inversores} ManyToOne*/ Catalogo_Inversores;
}
export { Transaction_Contratos_Inversionistas }
